src/validateEmail.test.js

```diff
@@ -8,13 +8,30 @@ describe(`Function 'validateEmail':`, () => {
   });
 
   it(`should return boolean`, () => {
+    expect(typeof validateEmail('goodEmail@.com')).toBe('boolean');
+  });
 
+  it(`should return 'true' for the valid email`, () => {
+    expect(validateEmail('test838@gmail.com.')).toBeTruthy();
+    expect(validateEmail('userName.userSurname@gmail.com.')).toBeTruthy();
+    expect(validateEmail('anabella123Syphora@g.com.')).toBeTruthy();
+    expect(validateEmail('SirkoIvan@uk.com')).toBeTruthy();
   });
 
   it(`should return 'true' for the valid email`, () => {
```

Comment:
The description for this test case is incorrect. It should indicate that it is testing for invalid emails, not valid ones.

Reply:
done